src/helpers/helpers.go

```diff
@@ -180,39 +180,14 @@ func GetDirContents(dirname string) []os.FileInfo {
 	return files
 }
 
-// GitInit attempts to initialize an empty git repository in the current
-// directory, or exits on failure.
-func GitInit() {
-	gitcmd := exec.Command("git", "init")
-	gitcmd.Stdout = os.Stdout
-	err := gitcmd.Run()
+// Git runs git with arguments and exit in case of failure.
```

Comment:
+1 on the code, but can we extend this comment to say args must be validated before being passed to this command? Or that any input from an external source MUST be validated and escaped correctly, so that I can't do something like:
"git status; rm -rf .*" if it's receiving *args* that aren't hardcoded (like they are here).

Reply:
Note that issue like one you mentioned, using ";", doesn't happen because exec package doesn't go through the shell to execute the command. It just exec the "git" directly and pass everything else as arguments for it (including the ";").